web/src/components/views/ProjectContainer.jsx

```diff
@@ -1,70 +1,105 @@
 import React, { Component } from 'react';
+import {fetchProjectDetails, fetchJson} from '../common/Backend'
 
-class ProjectContainer extends Component {
+export default class ProjectContainer extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      state:null
+      projectInf: []
     };
   }
 
+  componentWillReceiveProps(nextProps){
+    this.loadProjectInf(nextProps)
+  }
+
+  componentDidMount(){
+    this.loadProjectInf(this.props);
+  }
+
+  loadProjectInf(props) {
+    fetchProjectDetails(props.uuid).then(data => {
+      console.log(data);
+      this.setState({projectInf: data})
+    });
+  }
+
+
+
+
+  //
+
   render(){
+    const { _id, authors, date_creation, date_update, description } = this.state.projectInf;
+    const { status, tags, title, url} = this.state.projectInf;
+
+    var authors_string = null;
+    if (authors != null){
+      var author_container = []
+      for (var i = 0; i < authors.length; i++){
+        author_container.push(authors[i].name);
+      }
+      authors_string = author_container.join(", ")
+    } else {
+      authors_string = ''
+    }
+
+    var tag_string = null;
+    if (tags != null){
+      var tag_container = []
+      for (var i = 0; i < tags.length; i++){
```

Comment:
Since ES5 a bunch of cool and useful functionalities were added. Take a look at forEach() for example.